<?xml version="1.0" ?>
<launch>
    <param name="use_sim_time" value="true"/>
    <param name="analysing/test_name" value="ts1_t1"/>
    <param name="analysing/test_config" value="test_1"/>
    <param name="analysing/test_config_file" value="$(find atf_core)/config/test_config.yaml"/>
    <param name="analysing/result_yaml_output" value=""/>
    <param name="analysing/result_json_output" value="$(find atf_presenter)/data/"/>
    <test pkg="atf_core" test-name="test_analysing" time-limit="500.0" type="test_builder.py"/>
    <node args="--delay=5.0 --clock $(find atf_recorder)/data/ts1_t1.bag" name="player" output="screen" pkg="rosbag" type="play"/>
</launch>